src/xcode/ENA/ENA/Source/Scenes/Home/HomeCoordinator.swift

```diff
@@ -34,7 +34,7 @@ class HomeCoordinator: RequiresAppDependencies {
 
 	// MARK: - Internal
 
-	let rootViewController: UINavigationController = AppNavigationController(rootViewController: UIViewController())
+	let rootViewController: AppNavigationController = AppNavigationController(rootViewController: UIViewController())
```

Comment:
This change brings other functionality and configuration to the rooViewController, which can have side effects we don't see yet.
Would it be possible to separate the the "scroll up" functionality and use it without the use of AppNavigationController?

Reply:
@ArturFriesen Good point.
Actually, I wanted to participate in the private extensions for Array; but I can duplicate this code elsewhere.

Reply:
Instead of implementing the functionality in the `AppNavigationController`, I would rather move the extension out and make it reusable. A good thing would be: separate the responsibilities and keep the scope of `AppNavigationController` small.

Reply:
Please re-review @ArturFriesen. Thanks!